app.js

```diff
@@ -123,7 +78,13 @@ function dragOver(e) {
 
 function dragDrop(e) {
   e.stopPropagation(); // this prevents any funky behaviour, e.g. dropping two pieces and call this function twice
-  const correctTurn =
+  const correctTurn =  // TODO(martin): I would rename this one to isCorrectTurn.
+    // TODO(martin): Here you want to do `playerTurn`, not `"playerTurn"`.
+    //   So it should be:
+    //     draggedElement.firstChild.classList.contains(playerTurn);    -> no " " around playerTurn.
+    //   When you were trying to figure out your issue here with this code, it would be best if you peppered it with
+    //   console.log(...) calls and that way you could confirm is stuff is happening as expected, and then you would
+    //   realize that `correctTurn` is always false, whish is weird, and could investigate why is that.
```

Comment:
Aaah, thank you! It's still not dropping the pieces for some reason, but I'll look into it!
Good point about the console.log() - I'm aware of it and I tried to do it, but I'm not quite clear on what I'm supposed to console.log, so I leave it out. I'll try to get a better understanding of this.